### PR TITLE
Update GitHub and Wiki links

### PR DESCRIPTION
**Pull request by ShankarWarang**

### CODE REVIEW

File: packages/apps/src/Menu/index.tsx [hunk 1]
```diff
@@ -29,8 +29,10 @@ const disabledLog = new Map<string, string>();
 
 function createExternals (t: TFunction): ItemRoute[] {
   return [
-    { href: 'https://github.com/polkadot-js/apps', icon: 'code-branch', name: 'github', text: t<string>('nav.github', 'GitHub', { ns: 'apps-routing' }) },
-    { href: 'https://wiki.polkadot.network', icon: 'book', name: 'wiki', text: t<string>('nav.wiki', 'Wiki', { ns: 'apps-routing' }) }
+//    { href: 'https://github.com/polkadot-js/apps', icon: 'code-branch', name: 'github', text: t<string>('nav.github', 'GitHub', { ns: 'apps-routing' }) },
```
Comment (Depth-Hoar) [inline packages/apps/src/Menu/index.tsx:32]: I don't think we need to keep these is there a reason to keep them? I am talking about the commented out code //

**Review by Depth-Hoar — CHANGES_REQUESTED** [review]

File: packages/apps/src/Menu/index.tsx (hunk 1 above)
Comment (ShankarWarang) [reply packages/apps/src/Menu/index.tsx:32]: Renaming code without giving credits is something we should avoid. Commenting old code is somewhat like adding credits for source content?

**Review by Depth-Hoar — APPROVED** [review]

**Review by Depth-Hoar — APPROVED** [review]

File: packages/apps/src/Menu/index.tsx (hunk 1 above)
Comment (Depth-Hoar) [reply packages/apps/src/Menu/index.tsx:32]: Okay approved!

**Review by Depth-Hoar — APPROVED** [review]

File: packages/apps/src/Menu/index.tsx (hunk 1 above)
Comment (ShankarWarang) [reply packages/apps/src/Menu/index.tsx:32]: Cool, merging now.

**Pull request merged**